packages/create-sitecore-jss/src/common/utils/helpers.ts

```diff
@@ -32,11 +32,15 @@ export const getPascalCaseName = (name: string): string => {
  */
 export const openPackageJson = (pkgPath?: string) => {
   const filePath = path.resolve(pkgPath ?? `.${sep}package.json`);
+  return openJson(filePath);
+};
+
+export const openJson = (jsonPath: string) => {
```

Comment:
Instead of introducing another helper, can we just make the current `openPackageJson` and `writePackageJson` generic (suggested names: `openJsonFile` and `writeJsonFile`)?
I checked usage of these and in all cases the `pkgPath` is being passed. So, we could make the path parameter on the generic functions required like you've got.

Reply:
yea that makes sense, I will update that. Initially i was thinking to do that for _openPackageJson_ only.